src/service/fixed-layer.js

```diff
@@ -449,9 +453,19 @@ export class FixedLayer {
           // We shouldn't have too many of `fixed` elements.
           break;
         }
-        this.setupElement_(elements[j], fixedSelector, 'fixed');
+        const el = elements[j];
+        if (isInlineStylesEnabled) {
+          const sanitized = this.sanitizeElement_(el);
+          if (!hasInlineStyle && sanitized) {
+            hasInlineStyle = true;
+          }
+        }
+        this.setupElement_(el, fixedSelector, 'fixed');
       }
     }
+    if (isInlineStylesEnabled) {
+      user().error(TAG, 'Inline style not supported for fixed element');
```

Comment:
`"The style attribute is not supported yet for position-fixed elements."` And pass the element itself so it can be highlighted in dev console.